src/config/consts.js

```diff
@@ -22,19 +22,21 @@ export const consts = {
 
   MATOMO_TRACKING: {
     SCREEN_VIEW: {
-      HOME: 'Home',
-      NEWS_ITEMS: 'News',
-      EVENT_RECORDS: 'Events',
-      POINTS_OF_INTEREST: 'Points of interest',
-      TOURS: 'Tours',
-      POINTS_OF_INTEREST_AND_TOURS: 'Points of interest and tours',
-      SERVICE: 'Service',
+      BOOKMARK_CATEGORY: 'Bookmark Category',
```

Comment:
im englischen `category` klein

Reply:
changed it in 8a84e9e25ce56257a4c54ca67a967bcc32fcc310